packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -2687,7 +2688,9 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
                 minLines: widget.minLines,
                 expands: widget.expands,
                 strutStyle: widget.strutStyle,
-                selectionColor: widget.selectionColor,
+                // Special casing for SelectableText, show the selection even
+                // when the field isn't focused
+                selectionColor: (_hasFocus || widget.readOnly) ? widget.selectionColor : null,
```

Comment:
The `buildTextSpan` method has something similar (the composing region is not painted if the widget is `readOnly`). But this is a little bit too magical for a low level widget.

Reply:
Is it ever possible for a SelectableText to show its selection while another EditableText has focus?